charts/k8ssandra/templates/stargate/service_monitor.yaml

```diff
@@ -4,13 +4,15 @@ kind: ServiceMonitor
 metadata:
   name: {{ .Release.Name }}-prometheus-{{ include "k8ssandra.datacenterName" . }}-stargate
   labels:
+    release: {{ .Release.Name }}
+    app: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate
 {{ include "k8ssandra.labels" . | indent 4 }}
 spec:
   selector:
     matchLabels:
       app: {{ .Release.Name }}-{{ include "k8ssandra.datacenterName" . }}-stargate
   endpoints:
-  - port: metrics
+  - port: health
```

Comment:
I'd love to see an issue opened upstream to understand what the metrics port is used for. I get connection refused on it when port forwarding. That's not a blocker for this ticket, but seems like a good neighbor task

Reply:
That's the reason for this change. I'm not sure the metrics port is a thing anymore -- both health and metrics are served on port 8084. I am curious about why.